doc: Document DULT motion detector feature (#18138)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Author: I put it under `Integrations` section `DULT integration` subsection. Should I move it to the `Other libreries` section?

Reviewer: I'd say you might want to have something to both sections, DULT integration and Other libraries.

Author: I wonder what info should be put under integrations sections - now it is empty for all modules.

Author: For now I'll move it to other libraries and leave integrations section empty.

Reviewer 2: integration section is about the documentation updates

    * An application versioning using the :file:`VERSION` file.
    * The DFU support which can be enabled using the ``SB_CONFIG_APP_DFU`` sysbuild Kconfig option.
      DFU is available for all supported targets except the ``debug`` configurations of :ref:`zephyr:nrf52dk_nrf52832` and :ref:`zephyr:nrf52833dk_nrf52833` due to size constraints.

>     * An application module for the DULT motion detector feature and the new UI for supported board targets to interact with this feature.

Reviewer: Remove the changelog from this PR and consult @divipillai . She will add the entries to her PR.

Author: Yes, I just wanted to get approves here and later move it to Divya's PR.

Reviewer 3: Please remove DNM label when it's done, it could've been merged by accident.

Author: Done

      If this option is enabled, the Fast Pair subsystem tracks the Bluetooth bonds created through the Fast Pair Procedure and unpairs them if the procedure is incomplete or the Account Key associated with the bonds is removed.
      It also unpairs the Fast Pair Bluetooth bonds on Fast Pair factory reset.
      The option is enabled by default for Fast Pair use cases that are selected using :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_INPUT_DEVICE` and :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_MOUSE` Kconfig options.
    * The DULT motion detector feature integration in the FMDN extension.

>       It can be enabled using :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` Kconfig option.

Reviewer: ```suggestion
      You can enable this feature using the  :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` Kconfig option.
```

  * Added support for the nRF54L15 SoC.

* :ref:`dult_readme` library:

>   * Added DULT motion detector feature integration.

Reviewer 2: Nit: perhaps we should avoid the word integration as integration can occur at the higher level of the stack (at the FMDN extension level / at the sample level)

DULT integration
----------------

* Added DULT motion detector feature integration.

>   It can be enabled using :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` Kconfig option.

Reviewer: ```suggestion
  You can enable this feature using the :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` Kconfig option.
```
